server/api/projects/actions.py

```diff
@@ -126,3 +127,103 @@ def post(self, project_id):
             error_msg = f"Send message all - unhandled error: {str(e)}"
             current_app.logger.critical(error_msg)
             return {"Error": "Unable to send messages to mappers"}, 500
+
+
+class ProjectsActionsFeatureAPI(Resource):
+    @tm.pm_only(True)
+    @token_auth.login_required
+    def post(self, project_id):
```

Comment:
Based on AP redesign discussion - this should be a PATCH request

Reply:
Have we agreed on a consistent way of calling `actions` endpoints? They are different than the REST ones. I would propose to use `POST` for all of them. We also seem to use `PUT` on some others. A well defined way of selecting it, would be good.